ts/interactions/conversations/unsendingInteractions.ts

```diff
@@ -301,33 +290,35 @@ export async function deleteMessagesFromSwarmAndMarkAsDeletedLocally(
       'deleteMessagesFromSwarmAndMarkAsDeletedLocally: some messages failed to be deleted but still removing the messages content... '
     );
   }
-  await Promise.all(
-    messages.map(async message => {
-      return deleteMessageLocallyOnly({ conversation, message, deletionType: 'markDeleted' });
-    })
-  );
+  await deletesMessageLocallyOnly({ conversation, messages, deletionType: 'markDeleted' });
 }
 
 /**
  * Deletes a message completely or mark it as deleted only. Does not interact with the swarm at all
  * @param message Message to delete
  * @param deletionType 'complete' means completely delete the item from the database, markDeleted means empty the message content but keep an entry
  */
-export async function deleteMessageLocallyOnly({
+async function deletesMessageLocallyOnly({
```

Comment:
Thin you put the `s` in the wrong place. delete`s`MessageLocallyOnly -> deleteMessage`s`LocallyOnly

Reply:
fixed